recipes/fsleyes/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "fsleyes" %}
+{% set version = "0.27.0" %}
+{% set sha256 = "ccb47207c3bd34166341f4565b8d2f8d49759cbc45243036a9e09fa707e579a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - render  = fsleyes.render:main
+    - fsleyes = fsleyes.main:main
+
+requirements:
+  build:
+    - python
+    - pip
+    - sphinx 1.6.*
+    - sphinx_rtd_theme 0.*
+    - mock 2.*
+  run:
+    - python
+    - deprecation >=1.*,<=2.*
+    - fslpy >=1.13.2,<2
+    - fsleyes-props >=1.6.2,<2
+    - fsleyes-widgets >=0.6,<2
+    - indexed_gzip >=0.7,<1
+    - jinja2 2.*
+    - matplotlib >=1.5,<3
+    - nibabel 2.*
+    - numpy 1.*
+    - pillow >=3.2.0,<6
+    - pyopengl >=3.1.0,<4
+    - pyparsing 2.*
+    - scipy >=0.18,<2
+    - six 1.*
+    - trimesh >=2.22.28,<3
+    - rtree 0.8.3
+    - wxpython <4.1
+    - xnat >=0.3.3,<1
+    - wxnatpy >=0.3.0,<1
+  run_constrained:
+    - indexed_gzip >=0.7,<1.*
+    - ipykernel 4.8.*
+    - ipython 5.*
+    - jupyter_client 5.2.*
+    - notebook 5.5.*
+    - pyzmq 17.*
+    - tornado 5.*
+    - dcm2niix 1.0.20180622
```

Comment:
Are such strict pinnings on all of the above really needed?

Reply:
The notebook stuff is a bit experimental, so I was hesitant to support future versions. A cursory against the latest versions of eerything seems to work ok though..